client/components/pullRequests.js

```diff
@@ -1,6 +1,12 @@
 import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import {fetchOpenPullRequests} from "../actions/index";
+const moment = require('moment-business-time');
```

Comment:
We're using ES6 import modules rather than Node's require convention.
Please update to use import.

Reply:
Thanks for the explanation :) Fixed